### ISSUE TITLE
Row percentages

### ISSUE THREAD

**Issue by proc1125** [tompollard/tableone#137]
Hi for categorical variables, it will be really useful to get row percentages option. Is this possible? Thank you!

**Comment by tompollard** [comment]
I agree, this would be an improvement. Yes, definitely possible! I'll try to take a look.

**Comment by tompollard** [comment]
Sorry, no, but it is on my mind. I'm hoping to spend some time on updates in the next week or so, once I get some other things out of the way!
